config/sidekiq.yml

```diff
@@ -3,15 +3,15 @@
 :concurrency: 32
 :logfile: ./log/sidekiq.json.log
 :timeout: 4
-:queues:
-  - delivery_transactional
-  - [delivery_immediate_high, 10]
-  - [delivery_immediate, 5]
-  - [process_and_generate_emails, 4]
-  - [default, 3]
-  - [delivery_digest, 2]
-  - [email_generation_digest, 1]
-  - cleanup
+:queues: # See https://github.com/mperham/sidekiq/wiki/Advanced-Options#queues
+  - [delivery_transactional, 4]
+  - [delivery_immediate_high, 4]
```

Comment:
Do we want these two to be 10 still so that, if there are delivery_immediate_high, it does them at a more rapid rate?

As I understand here we've switched from checking delivery_immediate_high from 10 times as frequent as cleanup to only 4 times so would mean they end up less prioritised.

Reply:
Yeah, that's true. I was thinking the "10 times" setting was somewhat arbitrary, and it wasn't clear that "delivery_transactional" needed it as well. I think this indicates a general problem with using these weights, and I doubt we actually need them.

Reply:
So here we are again! I think we should keep the numbers simple, unless there's a empirical reason why a specific number is needed. Perhaps a simpler formula would be to use powers of two, so it's clearer expressed as "twice as fast". How about that?

Reply:
Sure, that sounds good.

@thomasleese do you know the logic for the use of 10? was there anything particular?

Reply:
I think my reasoning was that 10 is double the normal priority queue weight (5), so would be picked up at twice the rate of that queue. Other than that I don't think there was any particular reason why 10 specifically was chosen.

We were seeing normal priority and high priority emails being picked at a similar rate so we wanted a number much higher than the normal priority weighting.

Reply:
Thanks, that's good to know. I've updated the last commit to use powers of 2 and explain the reasoning. Could you take another look and let me know what you think?